src/main/java/com/faforever/client/chat/UserInfoWindowController.java

```diff
@@ -167,6 +171,19 @@ public void initialize() {
     nameColumn.setCellValueFactory(param -> param.getValue().nameProperty());
     changeDateColumn.setCellValueFactory(param -> param.getValue().changeDateProperty());
     changeDateColumn.setCellFactory(param -> new OffsetDateTimeCell<>(timeService));
+
+    timePeriodComboBox.getItems().addAll(i18n.get("userInfo.ratingHistory.allTime"),
```

Comment:
Maybe this would belong in enums. Imagine somebody changed the language in between then you would match against the wrong translations right?

Reply:
I know that is a little unlikely but idk still would like it better

Reply:
I changed it to use enums and use the selection model to get the index directly